custom_components/battery_notes/data/library.json

```diff
@@ -271,6 +276,11 @@
             "battery_type": "AA",
             "battery_quantity": 4
         },
+        {
+            "manufacturer": "by frient A/S",
```

Comment:
Is the leading "by " definitely there ?

Reply:
At least here:
<img width="197" alt="image" src="https://github.com/andrew-codechimp/HA-Battery-Notes/assets/3684954/b379524c-c0b7-4ceb-b72e-b3ceb85c85ea">

Reply:
Ah sorry, my bad... one sec...

Reply:
Thanks, easy mistake to make copy/pasting.